<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../qrs/resources.qrc">
    <normaloff>:/Staccato.ico</normaloff>:/Staccato.ico</iconset>
  </property>
  <property name="locale">
   <locale language="Portuguese" country="Brazil"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="styleSheet">
       <string notr="true">QTabBar::tab:disabled {
	 width: 0;
	 height: 0;
	 margin: 0;
	 padding: 0;
	 border: none; 
}

QTabWidget::pane {
	margin: 0px 0px 0px 0px; 
}
</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabOrcamentos">
       <attribute name="title">
        <string>Orçamentos</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="WidgetOrcamento" name="widgetOrcamento" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabVendas">
       <attribute name="title">
        <string>Vendas</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_20">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="WidgetVenda" name="widgetVenda" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabCompras">
       <attribute name="title">
        <string>Compras</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_21">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="TabCompras" name="widgetCompra" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabLogistica">
       <attribute name="title">
        <string>Logística</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="TabLogistica" name="widgetLogistica" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabNFe">
       <attribute name="title">
        <string>NF-e</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_17" columnminimumwidth="0,0">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="1">
         <widget class="TabNFe" name="widgetNFe" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabEstoque">
       <attribute name="title">
        <string>Estoque</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_22">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="TabEstoque" name="widgetEstoque" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabGalpao">
       <attribute name="title">
        <string>Galpão</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_8">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="TabGalpao" name="widgetGalpao" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabFinanceiro">
       <attribute name="title">
        <string>Financeiro</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_36">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="TabFinanceiro" name="widgetFinanceiro" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabRelatorios">
       <attribute name="title">
        <string>Relatórios</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="WidgetRelatorio" name="widgetRelatorio" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabGraficos">
       <attribute name="title">
        <string>Gráfico</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="WidgetGraficos" name="widgetGraficos" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabRh">
       <attribute name="title">
        <string>RH</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="WidgetRh" name="widgetRh" native="true">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabConsistencia">
       <attribute name="title">
        <string>Consistência</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_7">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="WidgetConsistencia" name="widgetConsistencia" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>764</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpcoes">
    <property name="title">
     <string>Opções</string>
    </property>
    <widget class="QMenu" name="menuTema">
     <property name="title">
      <string>Tema</string>
     </property>
     <addaction name="actionClaro"/>
     <addaction name="actionEscuro"/>
    </widget>
    <widget class="QMenu" name="menuImportar_tabela_fornecedor">
     <property name="title">
      <string>Importar tabela fornecedor</string>
     </property>
     <addaction name="actionProdutos"/>
     <addaction name="actionPromocao"/>
    </widget>
    <addaction name="actionConfiguracoes"/>
    <addaction name="menuImportar_tabela_fornecedor"/>
    <addaction name="actionImportar_tabela_IBPT"/>
    <addaction name="menuTema"/>
   </widget>
   <widget class="QMenu" name="menuAdministracao">
    <property name="title">
     <string>Administração</string>
    </property>
    <addaction name="actionGerenciar_dados_bancarios"/>
    <addaction name="actionCadastrarFornecedor"/>
    <addaction name="actionGerenciar_Lojas"/>
    <addaction name="actionGerenciar_NCMs"/>
    <addaction name="actionGerenciar_pagamentos"/>
    <addaction name="actionGerenciar_preco_estoque"/>
    <addaction name="actionCadastrarProdutos"/>
    <addaction name="actionCadastrarProfissional"/>
    <addaction name="actionGerenciar_staccatoOff"/>
    <addaction name="actionGerenciar_Transportadoras"/>
    <addaction name="actionCadastrarUsuario"/>
   </widget>
   <widget class="QMenu" name="menuVenda">
    <property name="title">
     <string>Venda</string>
    </property>
    <addaction name="actionCriarOrcamento"/>
    <addaction name="actionCadastrarCliente"/>
   </widget>
   <widget class="QMenu" name="menuAjuda">
    <property name="title">
     <string>Ajuda</string>
    </property>
    <addaction name="actionSobre"/>
   </widget>
   <widget class="QMenu" name="menuCalculadora">
    <property name="title">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../qrs/resources.qrc">
      <normaloff>:/calculator-icon.png</normaloff>:/calculator-icon.png</iconset>
    </property>
    <addaction name="actionCalculadora"/>
    <addaction name="actionCalcular_frete"/>
   </widget>
   <addaction name="menuVenda"/>
   <addaction name="menuAdministracao"/>
   <addaction name="menuOpcoes"/>
   <addaction name="menuAjuda"/>
   <addaction name="menuCalculadora"/>
  </widget>
  <action name="actionCriarOrcamento">
   <property name="text">
    <string>Criar orçamento</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionCadastrarProdutos">
   <property name="text">
    <string>Gerenciar produtos</string>
   </property>
  </action>
  <action name="actionCadastrarCliente">
   <property name="text">
    <string>Gerenciar cliente</string>
   </property>
   <property name="toolTip">
    <string>Cadastrar cliente</string>
   </property>
  </action>
  <action name="actionCadastrarUsuario">
   <property name="text">
    <string>Gerenciar usuários</string>
   </property>
   <property name="toolTip">
    <string>Cadastrar usuário</string>
   </property>
  </action>
  <action name="actionCadastrarProfissional">
   <property name="text">
    <string>Gerenciar profissionais</string>
   </property>
   <property name="toolTip">
    <string>Cadastrar profissional</string>
   </property>
  </action>
  <action name="actionGerenciar_Transportadoras">
   <property name="text">
    <string>Gerenciar transportadoras</string>
   </property>
  </action>
  <action name="actionGerenciar_Lojas">
   <property name="text">
    <string>Gerenciar lojas</string>
   </property>
  </action>
  <action name="actionCadastrarFornecedor">
   <property name="text">
    <string>Gerenciar fornecedores</string>
   </property>
   <property name="toolTip">
    <string>Cadastrar fornecedor</string>
   </property>
  </action>
  <action name="actionSobre">
   <property name="text">
    <string>Sobre o ERP Staccato</string>
   </property>
  </action>
  <action name="actionClaro">
   <property name="text">
    <string>Claro</string>
   </property>
  </action>
  <action name="actionEscuro">
   <property name="text">
    <string>Escuro</string>
   </property>
  </action>
  <action name="actionConfiguracoes">
   <property name="text">
    <string>Configurações</string>
   </property>
  </action>
  <action name="actionCalculadora">
   <property name="icon">
    <iconset resource="../qrs/resources.qrc">
     <normaloff>:/calculator-icon.png</normaloff>:/calculator-icon.png</iconset>
   </property>
   <property name="text">
    <string>Calculadora</string>
   </property>
  </action>
  <action name="actionProdutos">
   <property name="text">
    <string>Produtos</string>
   </property>
  </action>
  <action name="actionStaccatoOFF">
   <property name="text">
    <string>StaccatoOFF</string>
   </property>
  </action>
  <action name="actionPromocao">
   <property name="text">
    <string>Promoção</string>
   </property>
  </action>
  <action name="actionGerenciar_preco_estoque">
   <property name="text">
    <string>Gerenciar preço estoque</string>
   </property>
  </action>
  <action name="actionCalcular_frete">
   <property name="text">
    <string>Calcular frete</string>
   </property>
  </action>
  <action name="actionImportar_tabela_IBPT">
   <property name="text">
    <string>Importar tabela IBPT</string>
   </property>
  </action>
  <action name="actionGerenciar_NCMs">
   <property name="text">
    <string>Gerenciar NCMs</string>
   </property>
  </action>
  <action name="actionGerenciar_pagamentos">
   <property name="text">
    <string>Gerenciar pagamentos</string>
   </property>
  </action>
  <action name="actionGerenciar_staccatoOff">
   <property name="text">
    <string>Gerenciar STACCATO OFF</string>
   </property>
  </action>
  <action name="actionGerenciar_dados_bancarios">
   <property name="text">
    <string>Gerenciar dados bancários</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>WidgetVenda</class>
   <extends>QWidget</extends>
   <header>src/widgetvenda.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetOrcamento</class>
   <extends>QWidget</extends>
   <header>src/widgetorcamento.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TabCompras</class>
   <extends>QWidget</extends>
   <header>src/tabcompras.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TabLogistica</class>
   <extends>QWidget</extends>
   <header>src/tablogistica.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TabNFe</class>
   <extends>QWidget</extends>
   <header>src/tabnfe.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TabEstoque</class>
   <extends>QWidget</extends>
   <header>src/tabestoque.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetRelatorio</class>
   <extends>QWidget</extends>
   <header>src/widgetrelatorio.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TabFinanceiro</class>
   <extends>QWidget</extends>
   <header>src/tabfinanceiro.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetGraficos</class>
   <extends>QWidget</extends>
   <header>src/widgetgraficos.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetRh</class>
   <extends>QWidget</extends>
   <header>src/widgetrh.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetConsistencia</class>
   <extends>QWidget</extends>
   <header>src/widgetconsistencia.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TabGalpao</class>
   <extends>QWidget</extends>
   <header>src/tabgalpao.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qrs/resources.qrc"/>
 </resources>
 <connections/>
</ui>
